tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -432,6 +440,13 @@ public function testNativeArrayListSupport()
         $data = $stmt->fetchAll(PDO::FETCH_NUM);
         $this->assertEquals(5, count($data));
         $this->assertEquals(array(array(100), array(101), array(102), array(103), array(104)), $data);
+
+        $stmt = $this->_conn->executeQuery('SELECT test_int FROM fetch_table WHERE test_binary IN (?)',
+            array(array($binaryValue)), array(Connection::PARAM_STR_ARRAY));
```

Comment:
Ah now this makes a little more sense. Let me give this a try and see if it helps any.